eng/scripts/Language-Settings.ps1

```diff
@@ -278,7 +278,7 @@ function SourcePackageHasComFolder($artifactNamePrefix, $packageDirectory) {
 
     if ($LASTEXITCODE) {
       LogWarning "Could not download source artifact: $packageArtifact"
-      $mvnResults | Write-Host
+      $mvnResults | Write-Debug
```

Comment:
$mvnResults will be empty because the command is getting piped to `Out-Null`.

Reply:
Remove Out-Null and write-host when failed